two_lists_into_a_dict.py

```diff
@@ -4,5 +4,8 @@
 def convert_input_lists_into_dict(keys_list: List[str], values_list: List[int]) -> Dict[str, int]:
     converted_dictionary = {}
     for position in range(len(keys_list)):
+        if len(values_list) == position:
+            values_list.append(0)
```

Comment:
that part is not intuitive and to bo honest I don't understand what it's for?

Reply:
ahh is it the default value?

Reply:
Not really useful if you cannot provide it. How about raising an error if lengths of both list doesn't match?

Reply:
it resolves problem in situation with second list is empty or shorter than first one

Reply:
Fixed for now, I will create error rising as one of next tasks